docs/vsphere_ci/build.json

```diff
@@ -1,9 +1,9 @@
 {
   "variables": {
-    "os-iso-path": "[WorkloadDatastore] windows-iso-images/winsrv2004_sep_2020_x64_dvd.iso",
+    "os-iso-path": "[WorkloadDatastore] windows-iso-images/winsrv2022_sep_2021_x64_dvd.iso",
     "vmtools-iso-path": "[WorkloadDatastore] windows-iso-images/vmtools-v11360-windows.iso",
     "vm-template-folder": "windows-golden-images",
-    "vm-template-name": "windows-server-2004-template",
+    "vm-template-name": "windows-server-2022-template-noDocker",
```

Comment:
`noDocker`  is the new normal, I suggest keeping it simple: `windows-server-2022-template`

Reply:
Alright, I can do this and make the one to be used in release-4.10 `windows-server-2022-template-withDocker`